### PR TITLE
KAA-299: Implement an alternative option that would allow the user to populate the notification message data by filling in a form generated by avro-ui.

### PR DESCRIPTION
**Pull request by ikulikov**

### CODE REVIEW

File: server/common/server-shared/src/main/java/org/kaaproject/kaa/server/common/Version.java [hunk 1]
```diff
@@ -27,10 +27,10 @@
 public final class Version {
 
     /** The Constant PROJECT_VERSION. */
-    public static final String PROJECT_VERSION = "N/A";
+    public static final String PROJECT_VERSION = "0.7.0-SNAPSHOT";
```
Comment (ashvayka) [inline server/common/server-shared/src/main/java/org/kaaproject/kaa/server/common/Version.java]: No need to change this. Please revert this changes or add to .gitignore

**Comment by ashvayka** [comment]
reviewed

File: server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/place/SendNotificationPlace.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+        return true;
+    }
+
+    @Override
+    public boolean isLeaf() {
+        return true;
+    }
+
+    @Override
+    public TreePlaceDataProvider getDataProvider(EventBus eventBus) {
```
Comment (shpigor) [inline server/admin/src/main/java/org/kaaproject/kaa/server/admin/client/mvp/place/SendNotificationPlace.java]: I think it's better to move this implementation to abstract class TreePlace, and override it when needed.

**Comment by shpigor** [comment]
Reviewed
